<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuWindow</class>
 <widget class="QMainWindow" name="MenuWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MenuWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="pages">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>21</y>
      <width>1280</width>
      <height>677</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="main_page">
     <widget class="QLabel" name="Title">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>-30</y>
        <width>531</width>
        <height>381</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/resources/images/title.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="NewGame">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>280</y>
        <width>301</width>
        <height>81</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>301</width>
        <height>81</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/resources/images/New Game.png);
background-color: transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="LoadGame">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>360</y>
        <width>301</width>
        <height>81</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>301</width>
        <height>81</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/resources/images/Load Game.png);
background-color: transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="Exit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>440</y>
        <width>301</width>
        <height>81</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>301</width>
        <height>71</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/resources/images/Exit.png);
background-color: transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="title_diff">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>10</y>
        <width>301</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/resources/images/Difficulty.png);
background-color: transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
     </widget>
     <widget class="QPushButton" name="left_diff">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>150</y>
        <width>301</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="right_diff">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>150</y>
        <width>301</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="difficulty">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>150</y>
        <width>301</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="title_floors">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>290</y>
        <width>301</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/resources/images/Floors.png);
background-color: transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="left_floors">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>430</y>
        <width>301</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="floors">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>430</y>
        <width>301</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>50</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: gold;</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="right_floors">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>430</y>
        <width>301</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="StartGame">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>570</y>
        <width>301</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
